### PR TITLE
extend models (us, task, issue) and add epic model (init)

### PR DESCRIPTION
**Pull request by sopitz**
Hi @nephilaweb,
thanks for the python-taiga api.

This is my first time writing some python scripts and I would love to have some guidance of how to test my changes well.

I extended some models so it's more usable in tools and also added an epic-support.

Can you guide me along the way? I will update this pull request with the changes necessary to get this merged.

Thanks!
Cheers

### CODE REVIEW

**Comment by yakky** [comment]
@sopitz that's nice! Allow me a bit of time to review the code and provide further information on how to proceed. I'm really happy you decided to contribute

**Comment by sopitz** [comment]
@yakky AWESOME! thank you.

I've tried to set up a small slack bot for taiga. You can check it out here: https://github.com/sopitz/taigabot

**Comment by sopitz** [comment]
@yakky I have updated the pull request so travis is happy :)

**Comment by sopitz** [comment]
@yakky any news on this one?

**Comment by yakky** [comment]
@sopitz OMG, sorry! somehow I completely overlooked the notifications regarding this one!

File: taiga/models/models.py [hunk 1]
```diff
@@ -980,6 +997,25 @@ class Project(InstanceResource):
         'us_statuses': UserStoryStatuses
     }
 
+    def get_item_by_ref(self, ref):
+        response = self.requester.get(
+            '/resolver?project={project_id}&ref={task_ref}',
+            task_ref=ref,
```
Comment (yakky) [inline taiga/models/models.py]: just minor thing: can you rename this as ``obj_ref`` or anything more general than ``task_ref``?

**Pull request closed without merging**